setup.cfg

```diff
@@ -1,2 +1,2 @@
-[bdist_wheel]
-universal = 1
+;[bdist_wheel]
```

Comment:
We can delete this, we do not support Python 2 anymore.

Reply:
cool. I'll delete the file.